common/game_concepts/ancient_magic_concepts.txt

```diff
@@ -67,12 +68,24 @@ school_of_magic = {
     alias = { magic_school magical_school magic_discipline magical_discipline }
 }
 
+## Men At Arms
+stone_golems = {
+	texture = "gfx/interface/icons/regimenttypes/stone_golems.dds"
+    alias = { stone_golem }
+}
+
+lava_golems = {
+	texture = "gfx/interface/icons/regimenttypes/lava_golems.dds"
+    alias = { lava_golem }
+}
+
 undead = {
-    alias = { greater_undead }
+	texture = "gfx/interface/icons/regimenttypes/lesser_undead.dds"
+    alias = { lesser_undead greater_undead }
 }
 
 magocracy = {
-    alias = { magocracies magocratic magocracy_government }
+    alias = { magocracies magocratic magocratic_government }
```

Comment:
if changed then atleast alias = { magocracies magocratic magocratic_government magocracy_government }

Reply:
https://github.com/absylim/Ancient-Magic-CK3-Mod/blob/850f31ec7f63c69ac68505f59ddf74c7d7965b72/localization/english/ancient_magic_government_l_english.yml#L5

Note that this link is into your repo.

`magocracy_government` doesn't exist. `magocratic_government` does. That's why I changed it. I changed it to fix an error, I'm not going to add the error back in.